### ISSUE TITLE
Stuck on downloading profile.

### ISSUE THREAD

**Issue by xLoigi** [vopswtf/Fortify-Alpha#1]
![image](https://github.com/user-attachments/assets/b2c17992-4595-44b2-9c05-ecb7debf42ec)

I am stuck on downloading profiles.

**Comment by xLoigi** [comment]
![image](https://github.com/user-attachments/assets/5b249472-3fd8-4a5c-96c4-b40e8a5d9a33)
![image](https://github.com/user-attachments/assets/773e9f46-0648-4a0f-9ce6-2564da264b71)
Thats from the ingame console after the login attempt

**Comment by vopswtf** [comment]
sorry, anything from the backend's console?

**Comment by xLoigi** [comment]
![image](https://github.com/user-attachments/assets/fc1124d1-bd01-4fd8-aa4a-715238d59623)

**Comment by xLoigi** [comment]
![image](https://github.com/user-attachments/assets/f2cf2a0f-d521-42b2-8999-f6e27fe2e0e7)
Everytime i wait, it errors out.

**Comment by vopswtf** [comment]
could you send full log

**Comment by xLoigi** [comment]
From the backend or the game?

**Comment by vopswtf** [comment]
game

**Comment by vopswtf** [comment]
yeah i'm not really sure on this one, i've just tried with a fresh version of the repo with the same login
![image](https://github.com/user-attachments/assets/8f72502d-b266-4422-9ea7-7b07876bb6e7)

doesn't seem to be a backend issue but who knows

**Comment by xLoigi** [comment]
From where did you get the build?

**Comment by vopswtf** [comment]
another reason i might be getting a different result is because i'm using DefaultEngine.ini in config for redirection
![image](https://github.com/user-attachments/assets/843e3ce1-a336-4006-8ec5-7cf22fb89845)

**Comment by xLoigi** [comment]
Yeah that redirect fixed it. Thank you so much :)

**Comment by Toto0QI** [comment]
Hi im stuck here too, and after doing what u do I have the same error again : 
![image](https://github.com/user-attachments/assets/3fe0e0fe-f3a5-41a9-a95e-21976226fe5b)

**Comment by Toto0QI** [comment]
![image](https://github.com/user-attachments/assets/410745a5-1601-4b27-bb71-6f337ffddc9b)
after waiting I have this
